backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -682,6 +687,7 @@ export class ContributionResolver {
       .from(DbContribution, 'c')
       .leftJoinAndSelect('c.user', 'u')
       .where(`user_id = ${userId}`)
+      .withDeleted()
```

Comment:
Why this?

Reply:
Cause it should be seen in the admin interface (User List > User Details > CreationList)!

Reply:
Did you check this? As far as I understand this only applies to the deleted users but not to the deleted contributions

Reply:
Yeah in my eyes it worked on my local tests, but I will double check it!